Ascend.java

```diff
@@ -2,6 +2,20 @@
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
+		double lim = Double.parseDouble(args[0]);
+		int a,b,c;
+		a = (int)(lim*Math.random());

```

Comment:
Spaces between operations!
There are other places in the code where these spaces are missing.
Pay attention!